src/utils/helpers/loginCheck.ts

```diff
@@ -0,0 +1,6 @@
+const isLoggedIn = () => {
+  const JWT = localStorage.getItem('persistJWT')
```

Comment:
i think we need to check for either JWT being present or username in JWT = 'nonRegistered' (since even nonRegistered user will have persisted JWT,  can ask me why that is, I haven't documented that bit yet)

Reply:
Yeah, good point. And we should find a way to make sure we nuke the localStorage for the "nonRegistered" user after user registration form is submitted.

Reply:
Have added username to localStorage, but as we mentioned in meeting -- we need to allow 'nonRegistered' JWT to be considered "logged in", or it won't even be able to load the UserRego page.

Have made an issue to log out nonRegistered on Application Submit though. #150